geoips/compare_outputs.py

```diff
@@ -214,95 +214,46 @@ def images_match(output_product, compare_product, fuzz="5%"):
     bool
         Return True if images match, False if they differ
     """
-    out_diffimg = get_out_diff_fname(compare_product, output_product)
     exact_out_diffimg = get_out_diff_fname(
         compare_product, output_product, flag="exact_"
     )
-
-    call_list = [
-        "compare",
-        "-verbose",
-        "-quiet",
-        "-metric",
-        "ae",
-        "-fuzz",
-        fuzz,
-        output_product,
-        compare_product,
-        out_diffimg,
-    ]
-
-    exact_call_list = [
-        "compare",
-        "-verbose",
-        "-quiet",
-        "-metric",
-        "ae",
-        output_product,
-        compare_product,
-        exact_out_diffimg,
-    ]
-
-    LOG.info("**Running %s", " ".join(call_list))
-    fullimg_ret = subprocess.run(
-        call_list, stdout=subprocess.PIPE, stderr=subprocess.STDOUT
+    from PIL import Image
```

Comment:
Per conversation this morning, it is using pillow:

In [2]: import PIL

In [3]: PIL?
Type:        module
String form: <module 'PIL' from 'geoips/lib/python3.10/site-packages/PIL/__init__.py'>
File:        geoips/lib/python3.10/site-packages/PIL/__init__.py
Docstring:  
Pillow (Fork of the Python Imaging Library)

Pillow is the friendly PIL fork by Jeffrey A. Clark (Alex) and contributors.
    https://github.com/python-pillow/Pillow/

Pillow is forked from PIL 1.1.7.

PIL is the Python Imaging Library by Fredrik Lundh and contributors.
Copyright (c) 1999 by Secret Labs AB.

Use PIL.__version__ for this Pillow version.

;-)

Reply:
Yep, this looks good the way that it's working. Such an odd history here...